### DISCUSSION TITLE
New contact sensor added - not showing up in ring-mqtt

### DISCUSSION THREAD

**Discussion by harmgsn** [tsightler/ring-mqtt#704] [Support Q&A] [answered]
I've added a contact sensor to my Ring Alarm system.  It shows up in the Ring app without issue.  It will not show up as a device.  I've restarted the plugin, I've restarted HAOS... nothing.  I've been watching MQTT to see if it'll show up in there and it's not.  I've even forced a re-authentication to no avail so far.

**Comment by tsightler** [comment]
Logs from addon startup are the only way to have any chance to solve something like this, since that is when the discovered items are listed.

**Reply by harmgsn** [reply, marked as answer]
This was an MQTT broker issue, not an addon issue.  MQTT broker wasn't getting updates from ring.  I only found this after deleting the plugin and re-installing it... and seeing nothing update in MQTT.
